website/docs/cairo-1/03-project-initialization.md

```diff
@@ -90,60 +105,69 @@ the `protostar build` command and other commands to fail.
 
 ### Multi-contract project structure
 
-Each contract must be defined in the separate package: A different directory with separate `cairo_project.toml`
-and `lib.cairo` files defined.
+Each contract must be defined in the separate package: a different directory with separate `Scarb.toml`
+and `src/lib.cairo` files defined.
 
 ```
 my_project/
 ├── package1/
 │   ├── src/
-│   │   ├── contracts/
+│   │   ├── contract/
 │   │   │   └── hello_starknet.cairo
 │   │  ...
-│   │   ├── contracts.cairo
+│   │   ├── contract.cairo
 │   │   └── lib.cairo
-│   └── cairo_project.toml
+│   └── Scarb.toml
 ├── package2/
 │   ├── src/
-│   │   ├── contracts/
+│   │   ├── contract/
 │   │   │   └── other_contract.cairo
 │   │  ...
-│   │   ├── contracts.cairo
+│   │   ├── contract.cairo
 │   │   └── lib.cairo
-│   └── cairo_project.toml
+│   └── Scarb.toml
 ...
+├── src/
+│   └── lib.cairo
+├── Scarb.toml
 └── protostar.toml
 ```
 
-Make sure `[crate_roots]` are correctly defined.
-
-```toml title="package1/cairo_project.toml"
-[crate_roots]
-package1 = "src"
-```
+Notice that the whole project itself is a package too.
+This is due to the fact that [Scarb](https://docs.swmansion.com/scarb/), which Protostar uses 
+to manage dependencies, does not support workspaces yet. If you do not
+need to include any code in the top level package, just leave the `my_project/src/lib.cairo` file empty.
 
-```toml title="package2/cairo_project.toml"
-[crate_roots]
-package2 = "src"
-```
-
-Define each contract in the `[contracts]` section of the protostar.toml and each package
-in the `linked-librares`
+:::info 
+Even though `package1` and `package2` **directories** are inside `my_project` **directory**
+it does not make `package1` and `package2` **packages** parts of `my_project` **package**. 
+Therefore, you should refer to them using `package1::` and `package2::`.
+:::
 
```

Comment:
```suggestion
:::info 
:::
I don't think this is a correct abstraction. Everything imported to lib and its dependencies are parts of the package

```

Reply:
Yup, but those packages are not imported to lib, that's the point

Reply:
Yup but those are dependencies of those imported to lib